include/parse.h

```diff
@@ -58,24 +33,24 @@ NETPLAN_PUBLIC void
 netplan_parser_clear(NetplanParser **npp);
 
 NETPLAN_PUBLIC gboolean
-netplan_parser_load_yaml(NetplanParser* npp, const char* filename, GError** error);
+netplan_parser_load_yaml(NetplanParser* npp, const char* filename, NetplanError** error);
 
 NETPLAN_PUBLIC gboolean
-netplan_state_import_parser_results(NetplanState* np_state, NetplanParser* npp, GError** error);
+netplan_parser_load_yaml_from_fd(NetplanParser* npp, int input_fd, NetplanError** error);
 
 NETPLAN_PUBLIC gboolean
-netplan_parser_load_yaml_from_fd(NetplanParser* npp, int input_fd, GError** error);
+netplan_parser_load_nullable_fields(NetplanParser* npp, int input_fd, NetplanError** error);
 
 NETPLAN_PUBLIC gboolean
-netplan_parser_load_nullable_fields(NetplanParser* npp, int input_fd, GError** error);
+netplan_state_import_parser_results(NetplanState* np_state, NetplanParser* npp, NetplanError** error);
```

Comment:
It was purely aesthetic :) the netplan_state one was in the middle of a block of netplan_parser functions, I felt I should move it hehe. But I agree with your point, reduce the diff now is more important, it could be done later.